.github/workflows/contentcheck.yml

```diff
@@ -60,6 +60,11 @@ jobs:
         run: |
           exit `find astro/src/content/docs/ -type f -print|xargs grep -l 'icon:[a-z]'| wc -l |sed 's/[ ]*//g'`
         shell: bash
+      - name: Make sure docs don't use old asciidoc plus signs at end of line. Was used to extend line.
+        run: |
+          echo `grep -R '+$' astro/src/content/docs`
+          exit `find astro/src/content/docs/ -type f -print|xargs grep -l '+$'| wc -l |sed 's/[ ]*//g'`
+        shell: bash
```

Comment:
I don't know how I feel about this. 

If we are no longer coding in asciidoc, do we want to continue to run this? For example, if we run it once, fix up all of the stragglers, and we are no longer coding new asciidoc, is this still useful? 

In other words, any future `+` in theory are intended because we are not coding in Asciidoc so it doesn't seem like we should fail the build or linter check because of them. 

What if I want to add a literal `+` 😎 I don't know why.. but it doesn't seem like that should be illegal.

Reply:
I'm also concerned about adding any more of these to the github action instead of to locally-usable scripts or linting tools. I should be able to tell when I break something on my laptop before making a commit.

This is relatively opaque and it won't be clear to me, but especially less technical people, what went wrong and how to fix it.

Reply:
hmmm. let me see what I can do.